test/integration/repo/repo_test.rb

```diff
@@ -21,3 +21,16 @@
 #describe command('docker run hello-world') do
 #  its('exit_status') { should eq 0 }
 #end
+
+describe file('/var/repos/repos.key') do
+  it { should exist }
+  it { should be_symlink }
+end
+
+describe file('/var/repos/ubuntu/main/dists/focal/InRelease') do
+  it { should exist }
+end
+
+describe command('su - jenkins-agent -c "gpg --verify /var/repos/ubuntu/main/dists/focal/InRelease"') do
```

Comment:
Running this as the jenkins-agent user, whose keyring has the public and private signing keys on it, verifies that the signatures are all valid.

Reply:
Could be the case that this verifies that only a single signature is present and valid but we are not verifying in the PR that we are indeed signing the file with multiple keys?